force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Unused Code: Unnecessary code
#### Severity
Error

#### Finding
Method `finish` is not necessary. Consider removing it.

#### Why is this a problem?
Removing unnecessary code makes your app cleaner, more efficient and easier to understand.

#### Help and documentation

* [Irrelevant Code (CWE)](https://cwe.mitre.org/data/definitions/1164.html)